clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -597,6 +598,9 @@ do
   get_cw_package_checksums
   get_package_info
 
+  # Monit related files.
```

Comment:
It would be better if this lived with clearwater-monit, not clearwater-infrastructure (e.g. https://github.com/Metaswitch/chronos/blob/dev/chronos.root/usr/share/clearwater/clearwater-diags-monitor/scripts/chronos_diags).